taipy/gui/_renderers/factory.py

```diff
@@ -215,6 +215,8 @@ class _Factory:
                 ("width", PropertyType.string_or_number),
                 ("height", PropertyType.string_or_number),
                 ("hover_text", PropertyType.dynamic_string),
+                ("ref_id", PropertyType.dynamic_string),
```

Comment:
```suggestion
                ("anchor_id", PropertyType.dynamic_string),
```
?
Do you see a use case for dynamicity?

Reply:
Well your demo shows how useful this can be.
I'm in!